package.json

```diff
@@ -11,6 +11,7 @@
     "axios": "^0.21.1",
     "draft-js": "^0.11.7",
     "moment": "^2.27.0",
+    "package.json": "^2.0.1",
```

Comment:
Might have to fix this
Not a big issue, but was probably caused by using `npm install package.json` instead of `npm install`

Reply:
yeah it brought up a few security issues

Reply:
Oops 😶

Reply:
send a PR again by yeeting that xD